lib/Kanku/Notifier/NSCA.pm

```diff
@@ -75,6 +75,7 @@ sub notify {
   }
   my $nsca             = Net::NSCA::Client->new(%iopts);
   my $msg              = ($self->short_message || 'Got no message')."|duration=".$self->duration."s";
+  $msg .= "\n".$self->full_message."\n" if $self->full_message;
```

Comment:
Hmm, do you really want to send the full message?
This could be a lot of text

Reply:
it is the same text that you normally send in the mail and this is the same thing that I also did for other nsca notifications. I was expecting something between 1-200 lines, which should be okay.